src/app/item-popup/ItemPopupContainer.scss

```diff
@@ -239,13 +239,13 @@
 }
 
 .ammo-primary {
-  background-image: url('../../../destiny-icons/general/ammo_primary.svg');
+  background-image: url('~destiny-icons/general/ammo_primary.svg');
 }
 .ammo-special {
-  background-image: url('../../../destiny-icons/general/ammo_special.svg');
+  background-image: url('~destiny-icons/general/ammo_special.svg');
 }
 .ammo-heavy {
-  background-image: url('../../../destiny-icons//general/ammo_heavy.svg');
+  background-image: url('~destiny-icons//general/ammo_heavy.svg');
```

Comment:
```suggestion
  background-image: url('~destiny-icons/general/ammo_heavy.svg');
```

Reply:
ooh. hope that's it. it linted and ran fine for me locally after bhollis's fix before your tsconfig update so shrug